test/eslint.js

```diff
@@ -9,17 +9,18 @@ describe('node:eslint', function () {
       .on('end', done);
   });
 
-  it('creates .eslintrc', function () {
-    assert.file('.eslintrc');
-  });
-
-  it('fill env .eslintrc', function () {
-    assert.fileContent('.eslintrc', /"env": {/);
-    assert.fileContent('.eslintrc', /"node": true/);
-    assert.fileContent('.eslintrc', /"mocha": true/);
+  it('fill package.json', function () {
+    assert.fileContent('package.json', /"eslint-config-xo-space":/);
+    assert.JSONFileContent('package.json', { // eslint-disable-line new-cap
```

Comment:
Let's not add inline eslint changes. Let's disabled these rules if we don't follow them.